Test1/Test1/ThreadSafePriorityQueue.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Test1;
+
+public class ThreadSafePriorityQueue<P, V > where P : IComparable<P>
+{
+    private readonly SortedDictionary<P, List<V>> queue = new();
+    private int count = 0;
+    private readonly object queueLock = new();
+
+    public void Enqueue(P priority, V value)
```

Comment:
```suggestion
        /// <summary>
        /// Добавляет значение в очередь
        /// </summary>
    public void Enqueue(P priority, V value)
```
По-хорошему, каждый публичный метод, свойство и т.д. должен быть задокументирован.

Reply:
И да, порядок аргументов в задании другой.

Reply:
Отсутствие summary для публичных методов и свойств - просто комментарий.
Порядок аргументов - просто комментарий.